docs/getting-started/cluster-setup.md

```diff
@@ -0,0 +1,233 @@
+datastore-dynamodb        Opaque                                2         19d
+db-user-pass              Opaque                                2         49d
+default-token-abc55       kubernetes.io/service-account-token   3         50d
+git-token                 Opaque                                1         42d
+mysecret                  Opaque                                2         49d
+npm-token                 Opaque                                1         42d
+plugin-login-secrets      Opaque                                4         44d
+screwdriver-job-secrets   Opaque                                1         43d
+```
+
+The `<DEFAULT_TOKEN_NAME>` will be listed under `Name` when the `Type` is `kubernetes.io/service-account-token`.
+
+To get your k8s `token`, run:
+```bash
+$ kubectl describe secrets <DEFAULT_TOKEN_NAME>  # default-token-abc55 in this example
+```
+
+#### DynamoDB secrets
+To get your `accessKeyId` and `secretAccessKey`, navigate to [IAM](https://console.aws.amazon.com/iam)
+in your AWS console. Click on Users, and create a Screwdriver user with `<USERNAME>`. We recommend installing using an account which has full access to the AWS APIs. Select the `Security Credentials` tab and click `Create Access Key`. Download the file and keep note of those values.
+
+#### Screwdriver secrets
+You should create your own `default.yaml` file and fill out the constants with your own secrets.
+```yaml
+---
+login:
+    # A private key uses for signing jwt tokens. Can be anything
+    jwtPrivateKey: THIS-IS-AN-INSECURE-KEY
+    # The client id used for OAuth with github. Look up GitHub OAuth for details
+    # https://developer.github.com/v3/oauth/
+    oauthClientId: YOU-PROBABLY-WANT-SOMETHING-HERE
+    # The client secret used for OAuth with github
+    oauthClientSecret: AGAIN-SOMETHING-HERE-IS-USEFUL
+    # A password used for encrypting session, and OAuth data.
+    # **Needs to be minimum 32 characters**
+    password: WOW-ANOTHER-INSECURE-PASSWORD!!!
+    # A flag to set if the server is running over https.
+    # Used as a flag for the OAuth flow
+    https: false
+
+httpd:
+    port: 8080
+
+datastore:
+    plugin: dynamodb
+    imdb:
+        # File to read/write the database to
+        filename: ./database.json
+    dynamodb:
+        # AWS Access Key ID
+        accessKeyId: WHAT-A-LEGITIMATE-LOOKING-KEY-ID
+        # AWS Secret Access Key
+        secretAccessKey: TOTALLY-REAL-LOOKING-AWS-KEYS
+
+executor:
+    plugin: k8s
+    k8s:
+        # The host or IP of the kubernetes cluster
+        host: kubernetes
+        # The jwt token used for authenticating kubernetes requests
+        token: NOT-A-REAL-JWT-TOKEN
+
+webhooks:
+    github:
+        # Secret to add to GitHub webhooks so that we can validate them
+        secret: SUPER-SECRET-SIGNING-THING
+```
+
+Create the secret using `kubectl create`:
+```bash
+$ kubectl create -f ./secret.yaml
```

Comment:
This should probably actually say something like "Here are some secrets that you need, with these names:" followed by "Go here to learn how to make some secrets: http://kubernetes.io/docs/user-guide/secrets/" or "Here's an example of a secret.yaml, and here's how you run it"